terraform/modules/backend/.terraform.lock.hcl

```diff
@@ -2,64 +2,64 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/aws" {
-  version     = "5.9.0"
+  version     = "5.69.0"
```

Comment:
We could get rid of this from this folder instead of updating - there shouldn't be any reason to run terraform from it.